content/blog/2020-02-09-rstudio-conf-2020-what-they-forgot/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: What They Forgot to Teach You About R at rstudio::conf 2020
+author:
+  - "[Kara Woo](https://karawoo.com)"
+date: '2020-02-00'
+slug: rstudio-conf-2020-what-they-forgot
+categories: ["learn"]
+tags:
+  - rstudioconf
+photo:
+  url: https://unsplash.com/photos/okvqMfl78YE
+  author: Nielsen Ramon
+description: |
+  Highlights from the What They Forgot to Teach You About R workshop    
+---
+
+What did they forget to teach you about R? If you're like many people, your R
+education focused mainly on how to do statistical analyses in R. However, there
+are many other skills that are crucial to working productively in R and are
+rarely taught. Filling in these gaps was the focus of the What They Forgot to
+Teach You About R workshop that Jenny Bryan, Jim Hester, and I taught at
+rstudio::conf 2020.
+
+The first morning of the workshop was devoted to talking about
+[project-oriented workflows](https://rstudio-conf-2020.github.io/what-they-forgot/day1_1),
+and how best to organize one's R projects. We discussed how to create a project
+structure that is self-documenting, organized, and easy to revisit and
+collaborate on. Creating a logical folder structure, using informative and
+well-formatted file names, practicing "safe paths", and splitting analyses up
+into manageable pieces were some of the key components of robust R projects. In
+the afternoon, Jim led the group through an exploration of R's
+[debugging](https://rstudio-conf-2020.github.io/what-they-forgot/day1_3)
+tools, including `traceback()`, `browser()`, `debug()`, `trace()`, and
+`recover()`. Finally, we covered how to use
+[git and GitHub](https://rstudio-conf-2020.github.io/what-they-forgot/day1_4)
+to track the history of our projects and communicate our results.
+
+Day two began with [personal R administration](https://rstudio-conf-2020.github.io/what-they-forgot/day2_1):
+how to maintain your R installation and package libraries, how to set up a
+development environment, how to install source packages and read source code,
+and how to create reproducible project environments with renv. From there, Jenny
```

Comment:
Do we want a link to https://rstudio-conf-2020.github.io/what-they-forgot/day2_2 anywhere in here?

Reply:
Yeah I was thinking about that but wasn't sure where to put the link. Maybe I should make it "[personal R administration](https://rstudio-conf-2020.github.io/what-they-forgot/day2_1) ([part 2](https://rstudio-conf-2020.github.io/what-they-forgot/day2_2))"?

Reply:
yeah that sounds good